src/main/java/seedu/address/model/event/Time.java

```diff
@@ -40,9 +35,31 @@ public Time(String time) throws DateTimeParseException {
      * Returns true if a given string is a valid time.
      */
     public static boolean isValidTime(String test) {
```

Comment:
Similar to Date, this test could be done with `LocalTime.parse(test)`, catching the `DateTimeParseException` in the event of an invalid time provided by the user.

Reply:
The parse method accepts seconds (e.g. 10:15:15) and if we don't want to deal with that you can have a DateTimeParser.ofPattern("HH:mm").

Reply:
Nice! did not think of that, alr fixed the issue to use local time parser for validation